tests/journey/test_chamber_of_deputies_dataset.py

```diff
@@ -20,7 +20,7 @@ def setUp(self):
     def tearDown(self):
         rmtree(self.path, ignore_errors=True)
 
-
+    @skip('Takes to long to run')
```

Comment:
Is this just something to test the CI or is it supposed to go to `master`?

Reply:
This is just a test, looks that this test were making build fail using all memory available on Travis.